src/System.Security.Cryptography.X509Certificates/tests/System.Security.Cryptography.X509Certificates.Tests.builds

```diff
@@ -11,14 +11,22 @@
       <OSGroup>Unix</OSGroup>
     </Project>
     <Project Include="System.Security.Cryptography.X509Certificates.Tests.csproj">
+      <OSGroup>Unix</OSGroup>
+      <TargetGroup>netcoreapp1.1</TargetGroup>
```

Comment:
That is correct, netcoreapp1.1 is fine here. One Nit though, we have been trying to keep order of the metadata in the tests.builds files, so do you mind reordering it so that it is: TargetGroup, OSGroup, TestTFMs?

---
In reply to: [83250713](https://github.com/dotnet/corefx/pull/12610#discussion_r83250713) [](ancestors = 83250713)